moveit_setup_assistant/CMakeLists.txt

```diff
@@ -46,6 +46,16 @@ endif(NEW_YAMLCPP_FOUND)
 
 find_library(YAML yaml-cpp REQUIRED)
 
+catkin_package(
+  INCLUDE_DIRS
+    include
+  LIBRARIES
+    ${PROJECT_NAME}_tools
+  CATKIN_DEPENDS
+    moveit_ros_planning
+    moveit_ros_visualization
+)
+
```

Comment:
This is the only non-trivial change I did: I had to move the `catkin_package` command above the library declarations.
This is required because the resp. `add_library` and `set_target_properties` commands should stay near each other, but `${${PROJECT_NAME}_VERSION}` is defined by `catkin_package`.